ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/wildfly/extension/messaging-activemq/injection/main/module.xml

```diff
@@ -25,6 +25,7 @@
         <module name="jakarta.resource.api"/>
         <module name="jakarta.transaction.api"/>
 
+        <module name="org.jboss.as.cli"/>
```

Comment:
@soul2zimate @ehsavoie This makes me uncomfortable. How about the MessagingLogger methods just return String and the callers in the subsystem modules construct the OperationFormatException themselves?

There's a separation of concerns problem in this extension, where the 'subsystem' module is using the 'injection' module to hold its logger class. And of all the messages in the logger, only 1 is related to 'injection'! That's not clean but I'm not suggesting fixing that. Well, I am suggesting fixing that, just not necessarily here. :-)  But I don't think we should worsen the separation of concerns problem by bringing CLI types into 'injection'.

See also https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/Logger.20interface.20'duplication'/near/399726322

Reply:
@bstansberry Thank you for the review and suggestion. I have updated the fix.